File: plugin/octo_beat_plugin/ios/Classes/plugin/connection/OctoBeatConnectionCallback.swift
//
//  OctoBeatConnectionCallback.swift
//  octo_beat_plugin
//
//  Created by TRAN QUANG DAI on 10/02/2023.
//

import Foundation

protocol OctoBeatConnectionCallback {
    func onFoundOctoBeat(devices: [[String: String]])
    func onConnectedOctoBeat(device: [String: String])
    func onConnectFailOctoBeat()
}

File: plugin/octo_beat_plugin/android/src/main/kotlin/com/octo/octo_beat_plugin/plugin/connection/OctoBeatConnectionCallback.kt
package com.octo.octo_beat_plugin.plugin.connection

import java.util.ArrayList

interface OctoBeatConnectionCallback {
    fun onFoundOctoBeat(devices: ArrayList<Map<String, String>>)
    fun onConnectedOctoBeat(device: Map<String, String>)
    fun onConnectFailOctoBeat()
}